src/libopenrave-core/CMakeLists.txt

```diff
@@ -16,6 +16,13 @@ if( libpcrecpp_FOUND )
   add_definitions(-DHAVE_PCRECPP)
 endif()
 
+find_package(CURL)
+if ( CURL_FOUND )
+  set(OPENRAVE_CORE_LIBRARIES ${OPENRAVE_CORE_LIBRARIES} ${CURL_LIBRARIES})
+else()
+  message(FATAL_ERROR "curl library not found")
```

Comment:
@ziyan let's make it possible to use openrave without CURL. I would use `add_definitions` and check for that in the code

Reply:
Done.